examples/4_Sustainability/4-1_BoM_Sustainability_summary.py

```diff
@@ -49,320 +49,3 @@
     .with_units(mass=MASS_UNIT, energy=ENERGY_UNIT, distance=DISTANCE_UNIT)
 )
 # -
-
```

Comment:
Is this to address #687?

Reply:
No, that's just because deserialization of the summary type is broken. It's awkward, but possible, to add a mark to the dynamically-generated parameterization, but it's a lot more effort to avoid building the notebook. The options were:

* Comment out each line after this point. Not great, and easy to miss some uncomments because of the notebook format we use.
* Remove the reference to the notebook from sphinx, which should stop it building. You can't really do comments in ReST, so it's a deletion.

I went with deleting the problematic code. It's obvious that something's gone, so we won't forget to undelete it.